bower.json

```diff
@@ -6,12 +6,13 @@
     "expect": "0.2.x",
     "jquery": "1.9.x",
     "lodash": "2.3.x",
-    "backbone": "1.1.x",
+    "backbone": "git://github.com/jashkenas/backbone#3994c1cb8592c7e30613e9f869fe60c1cdce6164",
```

Comment:
Ah yeah we have to fix that. Should read >1.2.0 now

On Mon, May 18, 2015, 21:55 Paul Miller <notifications@github.com> wrote:

> In bower.json
> <https://github.com/chaplinjs/chaplin/pull/780#discussion_r30563667>:
>
> > @@ -6,12 +6,13 @@
> >      "expect": "0.2.x",
> >      "jquery": "1.9.x",
> >      "lodash": "2.3.x",
> > -    "backbone": "1.1.x",
> > +    "backbone": "git://github.com/jashkenas/backbone#3994c1cb8592c7e30613e9f869fe60c1cdce6164",
>
> ? Is it unstable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chaplinjs/chaplin/pull/780/files#r30563667>.
>